src/13_file_io.py

```diff
@@ -9,10 +9,17 @@
 # Print all the contents of the file, then close the file
 
 # YOUR CODE HERE
-
+foo_file = open("foo.txt", "r")
+foo_file.read()
```

Comment:
Check the provided documentation (linked in this file) to see how you can print the contents of the file you're reading as the instructions are stating.
![Screen Shot 2019-12-09 at 10 15 23 AM](https://user-images.githubusercontent.com/16110931/70447633-0a03cc80-1a6d-11ea-911c-744b7ceb2002.png)